java/distance-measurement-analysis/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,6 @@
 <resources>
-    <string name="app_name">distance-measurement-analysis</string>
+    <string name="app_name">Distance Measurement Analysis</string>
+    <string name="world_terrain_3d">http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer</string>
```

Comment:
Do you recommend any other string names for these files? 
I looked at the links but I could not come up with another more descriptive string name.